components/builder-db/src/metrics.rs

```diff
@@ -42,6 +46,10 @@ impl metrics::Metric for Histogram {
     fn id(&self) -> Cow<'static, str> {
         match *self {
             Histogram::DbCallTime => "db-call.call-time".into(),
+            Histogram::GetAllLatest => "db-call.call-time".into(),
```

Comment:
I believe you're going to want these to be distinct names

Reply:
ah dang you're correct, good catch